activate.sh

```diff
@@ -28,3 +28,8 @@ roles:
 EOF
 echo "master: localhost" > /etc/salt/minion.d/minion.conf
 echo "id: admin" > /etc/salt/minion.d/minion_id.conf
+
+# Disable swap
+# On ISO-based installs the script runs in a chroot and then the system is rebooted
+sed -i 's/^\(.* swap [ ]* sw .*\)$/#\1/' /etc/fstab
+sed -i 's/^\(.* swap [ ]* swap .*\)$/#\1/' /etc/fstab
```

Comment:
We could merge this two lines into
```
sed -i 's/^\(.* swap .*\)$/#\1/' /etc/fstab
```

Reply:
Or
```
sed -i '/ swap / s/^/#/' /etc/fstab
```

Reply:
I wanted to explicitly match ' swap ' followed by ' swap ' or ' sw ' (with whitespaces around them) in a more readable way.

Reply:
The partition type is always  swap ` with whitespaces, so we can search only for this string. Other options can be set in different way so it is not worth to worry about it.

Reply:
" swap " might potentially appear in a comment - that's why I was matching for the second sw/swap

Reply:
This will skip already commented lines with `swap` string inside
```
sed -i ' /^#/! / swap /s/^/#/' /etc/fstab
```

Problem which I see here is that these regexp are not covering following examples:
```
/dev/sda2 none swap sw,pri=1 0 0
```
or
```
/dev/sda3 none swap defaults 0 0
```